src/gene_to_reactions/app.py

```diff
@@ -23,12 +23,18 @@
 from venom.fields import MapField, String
 from venom.message import Message
 from gene_to_reactions.ice_client import IceClient
+import prometheus_client
```

Comment:
Isn't this a bit inconsistent that you explicitly import some things but you also import the library?

Reply:
Yes you're right. I didn't want to bring `CONTENT_TYPE_LATEST` directly into the namespace as it's not a very descriptive name but I can change that back.